reports/report.md

```diff
@@ -1672,13 +1674,60 @@ A deletion in a structure containing a mapping will not delete the mapping. The
 
 
 
+## H-30: Incorrect Modifier
+
+A modifier must contain a placeholder statement `_` and a `revert` or a `require` condition.
+
+<details><summary>6 Found Instances</summary>
+
+
+- Found in src/IncorrectModifier.sol [Line: 29](../tests/contract-playground/src/IncorrectModifier.sol#L29)
+
+	```solidity
+	    modifier onlyOwner() {
+	```
+
+- Found in src/IncorrectModifier.sol [Line: 38](../tests/contract-playground/src/IncorrectModifier.sol#L38)
+
+	```solidity
+	    modifier whenNotPaused() {
+	```
+
+- Found in src/IncorrectModifier.sol [Line: 46](../tests/contract-playground/src/IncorrectModifier.sol#L46)
+
+	```solidity
+	    modifier whenNotPaused2() {
+	```
+
+- Found in src/OnceModifierExample.sol [Line: 6](../tests/contract-playground/src/OnceModifierExample.sol#L6)
```

Comment:
Hmmm, This is just an empty modifier, but I don't think this detector should catch this one.

Reply:
For this one, I think it is should be caught because if a user disables the empty block detector, doesn't automatically mean they are okay with having a modifier that doesn't potentially revert when there is an undesired transaction. Also the empty detector is LOW, this one is HIGH

Reply:
Hmm I disagree on this one because looking into this description in Slither's Wiki, I am convinced we shouldn't have a modifier that just has a placeholder staement.

> If a modifier does not execute _ or revert, the execution of the function will return the default value, which can be misleading for the caller.

Reply:
I agree that placeholder modifiers are bad practice, but this is outside the scope of this detector. It should be a separate detector issue.

This detector is specifically intended to catch modifiers with a path inside it that does not include the `_;`, which this modifier does not adhere to.